astropy/modeling/core.py

```diff
@@ -1333,7 +1333,7 @@ def has_inverse(self):
         inverse defined.
         """
         try:
-            self.inverse
+            _ = self.inverse
```

Comment:
I am struggling to see the usefulness of B018 here and in the following cases – this looks like brushing over the _apparent_ uselessness of the expression (which is really required in this place) by assigning it to an (ultimately useless) variable. Could not find anything in PEP8 regarding this, though.

Reply:
Yeah I'm not also convinced that this improves readability

Reply:
I think either of these two would be an improvement over what is currently in the pull request.

Reply:
I also vote for `# noqa`

Reply:
Done!